wikipron/config.py

```diff
@@ -128,6 +130,41 @@ def _get_process_pron(
             processors.append(functools.partial(re.sub, r"\.", ""))
 
         prosodic_markers = frozenset(["ˈ", "ˌ", "."])
+        mods = frozenset(
+            [
+                "ˡ",
+                "ˍ",
+                "ʲ",
+                "ˠ",
+                "˺",
+                "ː",
+                "˞",
+                "˽",
+                "ˬ",
+                "˖",
+                "ʰ",
+                "ˤ",
+                "˳",
+                "˟",
+                "ⁿ",
+                "ʷ",
+                "˕",
+                "ˌ",
+                "˷",
+                "˔"
+            ]
+        )
+
+        def combining_mods(pron: str) -> str:
```

Comment:
This manipulates the combining modifiers but I'm not sure the name of the function is immediately explanatory. What about `split_combining_mod(ifiers)`?

Reply:
How about `_parse_combining_modifiers`, it seems a little more explanatory that way.